c7n/resources/s3.py

```diff
@@ -1424,7 +1424,7 @@ def process(self, buckets):
             return results
 
     def process_bucket(self, b):
-        p = b['Policy']
+        p = b.get('Policy')
```

Comment:
actually this change could be a little scary on second pass, this could potentially overwrite bucket policies as get_bucket_policy and put_bucket_policy are different iam permissions

Reply:
yeah we can come up with scenarios on it, but i think the key bit is the docs, we could also have filters checking denied methods ourselves, but its a bit paranoid, and can't account for `not` usage